lib/config.js

```diff
@@ -212,11 +235,7 @@ function parseED25519 (prefix) {
 
   let keyPair
   if (!ed25519.secret) {
-    if (process.env.NODE_ENV === 'production') {
-      throw new ServerError('No ' + (prefix ? prefix.toUpperCase() + '_' : '') +
-        'ED25519_SECRET_KEY provided.')
-    }
-    const seed = crypto.randomBytes(32)
+    const seed = generateSecret(prefix, 'ed25519')
```

Comment:
Since collisions in the `name` would be a huge disaster, I recommend putting a map in a central place and referencing it like:

``` js
const SECRET_NAMES = {
  ED25519: 'ed25519'
}
```

``` js
const seed = generateSecret(prefix, SECRET_NAMES.ED25519)
```

That way, when I create a new secret name and I go to add it to the map, I'll notice if there is a collision.